ui/page.go

```diff
@@ -722,21 +728,24 @@ func (page pageCommon) subpageHeader(gtx layout.Context, sp SubPage) layout.Dime
 			return page.theme.H6(sp.title).Layout(gtx)
 		}),
 		layout.Rigid(func(gtx layout.Context) layout.Dimensions {
-			return layout.Inset{Left: values.MarginPadding5, Top: values.MarginPadding5}.Layout(gtx, func(gtx C) D {
-				return decredmaterial.Card{
-					Color: page.theme.Color.Surface,
-				}.Layout(gtx, func(gtx C) D {
-					return layout.UniformInset(values.MarginPadding2).Layout(gtx, func(gtx C) D {
-						walletText := page.theme.Caption(sp.walletName)
-						walletText.Color = page.theme.Color.Gray
-						return walletText.Layout(gtx)
+			if sp.walletName == " " {
```

Comment:
This is a space and not an empty string. You should be checking if the walletName string is not empty.

Reply:
Apparently, testing with an empty string did not yield the desired result as the info icon and the `walletName` card was still displayed.
<img width="796" alt="image" src="https://user-images.githubusercontent.com/27733432/105883451-2a8f3d00-6007-11eb-89cd-9a765b9c5cf8.png">
 
 i also just noticed that the info icon and the wallet name was removed from all the other pages that had them when the space was added.

Explicitly defining `walletName : ""` and `infoTemplate : ""` did not resolve it either.
What seems to fix the issue are the `isWalletName` and `isInfoTemplate` variables that were initially added.